pkg/sqlite/load.go

```diff
@@ -921,36 +928,61 @@ func (s *sqlLoader) addBundleProperties(tx *sql.Tx, bundle *registry.Bundle) err
 		if err != nil {
 			return err
 		}
-		if err := s.addProperty(tx, registry.GVKType, string(value), bundle.Name, bundleVersion, bundle.BundleImage); err != nil {
-			return err
+		properties[propstring{Type: registry.GVKType, Value: string(value)}] = struct{}{}
+	}
+
+	// Add properties from annotations
+	csv, err := bundle.ClusterServiceVersion()
+	if err != nil {
+		// FIXME: Returning nil here is in line with the original implementation, but that was probably wrong. We should probably just bubble-up the error.
+		return nil
```

Comment:
As the FIXME indicates, we should return the error to avoid a hidden error that potentially drives people crazy during debugging.

Reply:
IIRC, the reason it wasn't changed is b/c we're not sure if folks are depending on this _not failing_.